src/Components/Layout/NavBar.js

```diff
@@ -19,43 +20,74 @@ class NavBar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      Navlink: ["Home", "Features", "Price", "Design"]
+      Navlink: [
+        {
+          name: "Home",
+          link: "/",
+        },
+        {
+          name: "Pricing",
+          link: "/pricing",
+        },
```

Comment:
Can we also temporarily add /admin ?
When we get closer to mvp we can fine tune if that should stay or not

Reply:
>Can we also temporarily add /admin ?

Okay I will just put it in the `state` but not add an item in the `Navbar` yet